README.md

```diff
@@ -1,11 +1,65 @@
 # Lightning Multiplexer
 
-This repository contains the code for the `lnmux` service. The problem that `lnmux` solves is fail-over for incoming Lightning payments. BOLT11 invoices are bound to a single node. If that node goes down, the invoice is no longer payable. The same is true for the node or its peers running out of inbound liquidity.
+This repository contains the code for Lightning Multiplexer service, `lnmux` for short.
+
+The problem that `lnmux` solves is fail-over for incoming Lightning payments. BOLT11 invoices are bound to a single node. If that node goes down, the invoice is no longer payable. The same is true for the node or its peers running out of inbound liquidity.
+
+A typical solution is to set up multiple nodes. If invoice expiry durations are short and down time is planned, it is possible to gracefully spin down a node by waiting for all invoices generated by that node to reach a final state. During that time, new invoices should only be generated on the remaining nodes.
+
+This however does not work for unexpected down time. An invoice that may have been passed on to a third party already becomes impossible to settle.
```

Comment:
nit: is this a third party, or the payer of the invoice (second party)?

Reply:
Ah good point, this is written from the perspective of a custodian. Updated.

Reply:
Ah, that makes sense. Looks good!